crates/ruma-common/CHANGELOG.md

```diff
@@ -25,6 +25,8 @@ Improvements:
   * Add `RoomMessageEventContent::make_for_reply`
 * Stabilize support for event replacements (edits)
 * Add support for read receipts for threads (MSC3771 / Matrix 1.4)
+* HTML-relevant characters (<, >, etc) in plaintext replies are now escaped during creation of the
```

Comment:
I believe this should be in a "Bug fixes" section above "Breaking changes". You can look at version 0.10.0 to see what I mean.

Reply:
Also the `<` and `>` need to be enclosed in `` ` ``.